libretro/LibretroVulkanContext.cpp

```diff
@@ -143,23 +143,28 @@ void LibretroVulkanContext::CreateDrawContext() {
 }
 
 void LibretroVulkanContext::Shutdown() {
-	LibretroHWRenderContext::Shutdown();
```

Comment:
I think this is fine, but there might be an argument for keeping this, just putting it below the LOST_BACKBUFFER (just in case someone in the future adds more to `LibretroHWRenderContext::Shutdown()`.

-[Unknown]

Reply:
Added it back. I really want to clean up the rats nest that is those context subclasses. Do some proper oo and not just trampoline into the super class and handle all the weird api permutations in a single method.